hw/ip/otbn/doc/_index.md

```diff
@@ -623,13 +638,62 @@ Before being stored in SRAM, the instruction word with the attached Integrity Pr
 The scrambling is reversed on a read.
 
 The ephemeral memory scrambling key and the nonce are provided by the [OTP block]({{<relref "/hw/ip/otp_ctrl/doc" >}}).
-They are set once when OTBN block is reset, and changed whenever a secure wipe of the instruction memory is performed.
-(TODO: Link to it once we have specified secure wipe.)
+They are set once when OTBN block is reset, and changed whenever a [secure wipe]({{<relref "#design-details-secure-wipe">}}) of the instruction memory is performed.
 
 The Integrity Protection Code is checked on every memory read, even though the code remains attached to the data.
 A further check must be performed when the data is consumed.
 Detected integrity violations in the data memory raise a fatal `imem_error`.
 
+### Secure Wipe {#design-details-secure-wipe}
+
+In the following situations OTBN itself initiates a full secure wipe:
+* The lifecycle controller asks for it through its escalation signal.
+* A fatal alert is issued. 
+  In this case, a full secure wipe is performed as local action.
+
+Software can trigger a full state wipe by writing 3’b111 to SEC_WIPE (i.e. set all individual state wipe bits).
+
+#### Data Memory (DMEM) Secure Wipe
```

Comment:
Could we rename this and the IMEM one to something like "DMEM key rotation"? I think it would make the other text a bit clearer, avoiding repeating various types of "wipe", which gets a bit confusing.

Reply:
Hm, I'm not sure how much we want to mix implementation details (key rotation) with what it achieves (get rid of data).

Reply:
Hmm, maybe. The problem is that I got quite confused when reading through this patch and I'm supposed to know what's going on already! I think it would really help if the names were more distinct.